tools/install.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Since this file will be downloaded to the user's machine, could you add the [MIT License](https://github.com/davrodpin/mole/blob/master/LICENSE) to the beginning of the file?

Reply:
Sure.